ui/app/AppLayouts/Wallet/popups/swap/SwapInputParamsForm.qml

```diff
@@ -45,12 +47,9 @@ QtObject {
         return !!root.selectedAccountAddress &&
                 root.selectedNetworkChainId !== -1 &&
                 !!root.fromTokensKey && !!root.toTokenKey &&
-                ((!!root.fromTokenAmount &&
-                  !isNaN(SQUtils.AmountsArithmetic.fromString(root.fromTokenAmount)) &&
-                  SQUtils.AmountsArithmetic.fromString(root.fromTokenAmount) > 0) ||
-                 (!!root.toTokenAmount &&
-                  !isNaN(SQUtils.AmountsArithmetic.fromString(root.toTokenAmount)) &&
-                  SQUtils.AmountsArithmetic.fromString(root.toTokenAmount) > 0 )) &&
+                (!!root.fromTokenAmount &&
+                 !isNaN(SQUtils.AmountsArithmetic.fromString(root.fromTokenAmount)) &&
```

Comment:
Revisiting again, I don't think it's correct to call `isNaN` on a BigInt number

Reply:
Yeah, not sure if the `isNaN` works correctly on a Big number

Reply:
https://github.com/status-im/status-desktop/pull/15449/files#:~:text=(numStr)%20%7B-,function%20fromString(numStr)%20%7B,-Comment modified this function as per discussion with @micieslak as it was not correctly handling exceptions right now. and now it should return nan in case we pass an invalid value.